### PR TITLE
6329 Creating/Editing Rectangular Duct WAF data in UI

### LINKED ISSUE #6329: Update MP schema: numberOfTraversePointsWAF
In the MP schema, numberOfTraversePointsWaf needs to be updated to:

- numberOfTraversePointsWAF

**Acceptance tests:**

- Review of the JSON schema file shows the updated JSON element
- A file using the new JSON element name is able to be imported successfully
- Upon export, the exported file shows the element with the updated name
- Verify that the "lastUpdated" date field in the schema document is updated.

### PR DESCRIPTION
**Pull request by usmanahmederg**
## Ticket
https://github.com/US-EPA-CAMD/easey-ui/issues/6329

## Changes
Update new version of easey-common 17.5.1 for Swagger page example schema

### CODE REVIEW

**Review by ergjustin — DISMISSED** [review]

File: src/duct-waf-workspace/duct-waf.service.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export class DuctWafWorkspaceService {
       wafMethodCode: payload.wafMethodCode,
       wafValue: payload.wafValue,
       numberOfTestRuns: payload.numberOfTestRuns,
-      numberOfTraversePointsWaf: payload.numberOfTraversePointsWAF,
+      numberOfTraversePointsWaf: Number(payload.numberOfTraversePointsWAF),
```
Comment (ergjustin) [inline src/duct-waf-workspace/duct-waf.service.ts]: @usmanahmederg Is it necessary to convert `numberOfTraversePointsWAF` value to number?

File: src/dtos/duct-waf.dto.ts
Comment (ergjustin) [inline src/dtos/duct-waf.dto.ts:1]: @usmanahmederg  In swagger page the schema example is still in small letters.
image

Comment (usmanahmederg) [reply src/dtos/duct-waf.dto.ts:1]: @ergjustin  I've looked into it and this is coming from `Easey-Common` as well, we would need to make change in that repository

**Comment by ergjustin** [comment]
Could you please clarify the specific changes you are proposing in this pull request? Upon reviewing the previous and current versions, I do not notice any differences.

**Review by maxdiebold-erg — APPROVED** [review]
Looks good, this shows properly in Swagger, is editable via the UI, and exports with the proper name.

**Pull request merged**